mesonbuild/backend/ninjabackend.py

```diff
@@ -2098,7 +2098,10 @@ def _generate_single_compile(self, target, compiler, is_generated=False):
         # Hence, we must reverse the list so that the order is preserved.
         for i in reversed(target.get_include_dirs()):
             basedir = i.get_curdir()
-            for d in i.get_incdirs():
+            #We should iterate include dirs in reversed orders because
+            #-Ipath will add to begin of array. And without reverse
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->